src/app/pages/ArticlePage/ArticlePage.jsx

```diff
@@ -108,6 +108,8 @@ const ArticlePage = ({ pageData }) => {
   );
   const recommendationsData = pathOr([], ['recommendations'], pageData);
   const isPGL = pageData?.metadata?.type === 'PGL';
+  const isSTY = pageData?.metadata?.type === 'STY';
+  const isCPS = isPGL || isSTY;
```

Comment:
Is it worth using the const values from https://github.com/bbc/simorgh/blob/0ce46bb22abc9cc777d066b539ba6174be71c6cb/src/app/routes/utils/pageTypes.ts to get the page type?